tests/Type/GeneralTypeTest.php

```diff
@@ -6,7 +6,7 @@
 
 use PHPStan\Testing\TypeInferenceTestCase;
 
-use function version_compare;
+use function Orchestra\Testbench\laravel_version_compare;
```

Comment:
What does this function do differently?

Reply:
This function properly handles the `12.x-dev` version, otherwise you would have to do:

![image](https://github.com/user-attachments/assets/5fbc8f47-45a5-4606-b694-208f14f468f0)

Reply:
@crynobone This function is deprecated. Should we require sidekick here and use the function?

Reply:
Done